presto-main/src/main/java/io/prestosql/sql/planner/plan/JoinNode.java

```diff
@@ -52,6 +52,7 @@
     private final List<EquiJoinClause> criteria;
     private final List<Symbol> leftOutputSymbols;
     private final List<Symbol> rightOutputSymbols;
+    private final boolean cardinalityInsensitive;
```

Comment:
I think we could try to find a different name for this property.

"cardinality insensitive" suggests that the output of JoinNode is independent of cardinality of its inputs. This is true only for certain cases depending on implementation. Actually, this property captures a relation between the JoinNode and its parent.
I would rename it to "is in cardinality insensitive context", or simply "can skip duplicates".

Reply:
What about `outputCardinalityInsensitive` or `canSkipOutputDuplicates?

`can skip duplicates` also doesn't say if it's input or output

Reply:
`canSkipOutputDuplicates` is great.